src/ripple/app/misc/SHAMapStoreImp.h

```diff
@@ -190,7 +190,7 @@ class SHAMapStoreImp : public SHAMapStore
     {
         std::uint64_t check = 0;
 
-        for (uint256 it: cache.getKeys())
+        for (auto const& it: cache.getKeys())
```

Comment:
I've changed `it` to `key`.  I've also changed `auto` back to `uint256`, though this will be a temporary change (I hope).  I had originally changed it because I changed _one_ of the cache types here: https://github.com/HowardHinnant/rippled/blob/shamaphash/src/ripple/app/misc/SHAMapStoreImp.cpp#L556-L561  But then I had to back that change out, as it was too far reaching for phase 1.  We will delay the `uint256` / `auto` choice here until it is forced upon us by phase 2 or 3.